recipes/poppler-data/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class PopplerDataConan(ConanFile):
+    name = "poppler-data"
+    description = "encoding files for use with poppler, enable CJK and Cyrrilic"
+    homepage = "https://poppler.freedesktop.org/"
+    topics = "conan", "poppler", "pdf", "rendering"
+    license = "GPL-2.0-or-later", "adobe", "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("poppler-data-{}".format(self.version), self._source_subfolder)
+
+    @property
+    def _datadir(self):
+        return os.path.join(self.package_folder, "bin")
+
+    def _configure_cmake(self):
+        # FIXME: USE_CMS
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["datadir"] = self._datadir
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _patch_sources(self):
+        for patchdata in self.conan_data["patches"][self.version]:
+            tools.patch(**patchdata)
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING*", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self._datadir, "pkgconfig"))
+
+        # Create a dummy header to circumvent CI
+        tools.save(os.path.join(self.package_folder, "include", "_dummy.h"), "")
```

Comment:
Let me guess, CCI hook? If yes, we can open a PR to hooks instead.

Reply:
You guessed right :smile:

Reply:
Okay, I gonna open a PR and link to this thread.

Reply:
https://github.com/conan-io/hooks/pull/237

Reply:
Thanks!

Reply:
Whitelisted in hooks https://github.com/conan-io/hooks/pull/237

Reply:
@madebr now you can remove it!

Reply:
Aaaaand it's gone.